### ISSUE TITLE
Open-source for GitHub contributor stats (https://github.com/HwangTaehyun/github-contributor-stats)

### ISSUE THREAD

**Issue by HwangTaehyun** [anuraghazra/github-readme-stats#2027]
**Is your feature request related to a problem? Please describe.**

* It would be nice to have a card showing GitHub contributor stats so that we can show at a glance which open-sources we are interested in.

![Taehyun's GitHub Contributor stats](https://github-contributor-stats.vercel.app/api?username=HwangTaehyun&hide=B)

**Describe the solution you'd like**

* I make the open-source for GitHub contributor stats (https://github.com/HwangTaehyun/github-contributor-stats)
It is based on [github-readme-stats](https://github.com/anuraghazra/github-readme-stats), I focused on showing GitHub contributor stats and applied typescript to the original project. Thanks to [@anuraghazra](https://github.com/anuraghazra) for the awesome open-source project!

**Comment by rickstaa** [comment]
@HwangTaehyun Thanks for bringing your project to our attention, and great to see that you did your project in TS. 🚀 Would be great to migrate `github-readme-stats` over to typescript in the future.

I like your idea and think it is within the scope of the `github-readme-stats` repository. I would love to add it to the `github-readme-stats` repository if you are interested. Nevertheless, I have to check with @anuraghazra if he thinks this is a good idea since we are already rate-limited by GitHub. I proposed #1867 to fix this, but we still have to decide on this issue.

**Comment by HwangTaehyun** [comment]
@rickstaa Oh, I'm delighted to hear that! Thanks for your attention to this work and I'm looking forward to hearing from you and @anuraghazra!

**Comment by anuraghazra** [comment]
The name kind of is misleading tho. 

is it actually "top" or "latest"? 
It looks the most latest organizations I've contributed to.

**Comment by rickstaa** [comment]
> The name kind of is misleading tho.
> 
> is it actually "top" or "latest"? It looks the most latest organizations I've contributed to.

I agree with @anuraghazra. [Based on your code](https://github.com/HwangTaehyun/github-contributor-stats/blob/3b72a77c85c12ca4ddadef46662fe2965baf13c6/src/fetchContributorStats.ts#L34-L51) you are currently showing the latest contributions. A card I could see within the GRS family would one that would show your top contributions to other repositories. Meaning the repositories to which you made the most PR and commits. I am not sure if we can directly get this information using the current GraphQL API (see [repositoryconnection](https://docs.github.com/en/graphql/reference/objects#repositoryconnection)). I saw that some people could retrieve this information using two Graphql calls (see https://stackoverflow.com/questions/54757319/githubs-graphql-api-how-can-i-get-all-of-the-commits-ive-contributed-to-a-par).

**Comment by HwangTaehyun** [comment]
I intended to show all contributor stats with ranking order and some customized hide settings. I refer to your link and test it. However, maybe there are some problems when showing all contributions because GitHub told me that they offer only within 3-5 years contributions for graphql API when I asked.

**Comment by HwangTaehyun** [comment]
I want to know what the meaning of top contribution. Repo that i contributed by many PRs or Top stars repo though i contributed only once.
Currently, I intended to show latter repos!

**Comment by rickstaa** [comment]
@HwangTaehyun Ah, good point! I now understand your intentions with the card and why there is a score behind the repo. 😄 The card I had in mind would check all the repositories a user has contributed to, filter them according to the number of PR and list them from the most PR to the least. This way, a user can show in which repositories he is a frequent contributor.
